<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadWriteRegsDialog</class>
 <widget class="QDialog" name="ReadWriteRegsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read/Write registers</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="lineEdit_Address">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ValWrite">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ValRead">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
  </widget>
  <widget class="QLabel" name="label_Address">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Value">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Write">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Read">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Status">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>230</red>
         <green>230</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use &quot;0x&quot; prefix for hexadecimal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0x</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Write</sender>
   <signal>clicked()</signal>
   <receiver>ReadWriteRegsDialog</receiver>
   <slot>s_WriteReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Read</sender>
   <signal>clicked()</signal>
   <receiver>ReadWriteRegsDialog</receiver>
   <slot>s_ReadReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_WriteReg()</slot>
  <slot>s_ReadReg()</slot>
 </slots>
</ui>
